example/style.css

```diff
@@ -22,10 +22,14 @@ body {
   right: -30rem;
 }
 
-.js-accordion [aria-hidden=true] {
+.glossary div[aria-hidden=true] {
```

Comment:
Do we need the div part of the selector? What about:

`.glossary [aria-hidden=true] { `